src/main.cpp

```diff
@@ -186,6 +193,29 @@ void opcontrol()
 			rightSpeed = analogY;
 		}
 
+		if (master.get_digital(DIGITAL_A))
+		{
+			pros::c::adi_digital_write(PneumaticPort, HIGH);
+			ConveyorDirection = 1;
+			ConveyorOn = true;
+		}
+
+		if (master.get_digital(DIGITAL_B))
+		{
+			pros::c::adi_digital_write(PneumaticPort, HIGH);
+			ConveyorDirection = -1;
+			ConveyorOn = true;
+		}
+
+		if (master.get_digital(DIGITAL_X))
+		{
+			pros::c::adi_digital_write(PneumaticPort, LOW);
+			ConveyorDirection = 0;
+			ConveyorOn = false;
+		}
+
+		conveyor_motor.move(ConveyorSpeed * ConveyorDirection);
```

Comment:
Your code assumes we have a spare motor to use for conveyor and has no other usage for this motor in the code. With that, it's either incomplete, or usage of pneumatics is not needed (as there is no motor sharing in this code).
I think it's right direction for alternative design (where motor is shared between conveyor and lift), and I think (now) it's better way of sharing motor (over taking input from wheels), but it's not fully baked in your code.